views/apartments/index.erb

```diff
@@ -0,0 +1,26 @@
+<header
+<nav>
+  <a href="http://localhost:4567/">home</a>
+</nav>
+</header>
+
+<h1>
+  Available Apartments
+</h1>
+
+  <% @apartments.each do |apartment| %>
+<li>
+  <a href="">
+  <%= apartment.address %>
+</a>
+</li>
+<%end%>
+
+<form method='post' action='/apartments'>
+  <input type='text' address='address' placeholder="address">
```

Comment:
this is what's preventing the creation from working. should be `<input type='text' name='address' placeholder="address">`

The `name` attribute is what specifies the key in the params hash